src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -13,13 +13,14 @@
 from sentry.models import Integration, Project, RepositoryProjectPathConfig
 from sentry.shared_integrations.exceptions import ApiError
 from sentry.utils.event_frames import munged_filename_and_frames
+from sentry.utils.json import JSONData
 
 logger = logging.getLogger(__name__)
 
 
 def get_link(
     config: RepositoryProjectPathConfig, filepath: str, version: Optional[str] = None
-) -> Any:
+) -> Mapping[str, str]:
```

Comment:
I generally recommend "accept generic parameters, return concrete types" since it works better with variance rules (so I'd use `dict[str, str]` here but I'd use `Mapping[str, str]` as a parameter)

Reply:
That's a good idea!